src/PrestaShopBundle/Controller/Admin/TranslationsController.php

```diff
@@ -222,7 +222,7 @@ protected function getTranslationsCatalogue(Request $request)
 
         $locale = $this->langToLocale($lang);
 
-        if (!is_null($theme)) {
+        if (!is_null($theme) && 'themes' === $type) {
```

Comment:
yes, this is why you have to merge #6353 before, as noticed in PR description. I needed this fix to do the job here.